# --- T2-COPYRIGHT-NOTE-BEGIN ---
# This copyright note is auto-generated by ./scripts/Create-CopyPatch.
# 
# T2 SDE: package/.../xemacs-beta/no-info.info.patch
# Copyright (C) 2009 - 2017 The T2 SDE Project
# 
# More information can be found in the files COPYING and README.
# 
# This patch file is dual-licensed. It is available under the license the
# patched project is licensed under, as long as it is an OpenSource license
# as defined at http://www.opensource.org/ (e.g. BSD, X11) or under the terms
# of the GNU General Public License as published by the Free Software
# Foundation; either version 2 of the License, or (at your option) any later
# version.
# --- T2-COPYRIGHT-NOTE-END ---

--- xemacs-21.5.34/man/Makefile.vanilla	2017-03-18 21:09:29.816833106 +0000
+++ xemacs-21.5.34/man/Makefile	2017-03-18 21:10:02.608831761 +0000
@@ -263,7 +263,7 @@
 new-users-guide.info : $(INFODIR)/new-users-guide.info
 texinfo : $(INFODIR)/texinfo.info
 
-.PHONY : xemacs lispref internals new-users-guide texinfo info dvi pdf
+.PHONY : xemacs lispref internals new-users-guide info dvi pdf
 
 info : $(info_files)
 html : $(html_files)
